Assets/MixedRealityToolkit/_Core/Definitions/Utilities/CollationOrderTypeEnum.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities
+{
+    /// <summary>
+    /// Collation order type used for sorting
+    /// </summary>
+    public enum CollationOrderTypeEnum
+    {
+        /// <summary>
+        /// Don't sort, just display in order received
+        /// </summary>
+        None,
+        /// <summary>
+        /// Sort by transform order
```

Comment:
I haven't used this, so I'm not sure what "transform order" is, but this documentation doesn't help me understand. If you know what this order does, can you improve the summary?

Reply:
I'm not sure, I've never used it before - this is leftover from the original enum.

Reply:
It compares sibling index, I'll rename to better describe the behavior.